ipatests/pytest_ipa/integration/tasks.py

```diff
@@ -2056,3 +2056,16 @@ def uninstall_packages(host, pkgs):
     else:
         raise ValueError('install_packages: unknown platform %s' % platform)
     host.run_command(install_cmd + pkgs)
+
+
+def pkg_installer(host):
```

Comment:
Why define a new function rather than re-use tasks.install_packages() and tasks.uninstall_packages() ?

Reply:
Ubuntu scenario not covered there. 
I think i can modify tasks.install_pakcage()/uninstall_packge() . is that ok?